src/__tests__/utils.test.ts

```diff
@@ -160,6 +161,22 @@ describe('isStableRelease util', () => {
 	})
 })
 
+describe('sanitizeReleaseGroupTitle', () => {
+	it.each`
+		input                  | output
+		${'BREAKING CHANGES'}  | ${'breaking changes'}
+		${'Bug Fixes'}         | ${'bug fixes'}
+		${'minor'}             | ${'minor'}
+		${'minor-release'}     | ${'minor release'}
+		${'🐛 Bug fix'}        | ${'bug fix'}
```

Comment:
![emojis-headings](https://github.com/octoclairvoyant/octoclairvoyant-webapp/assets/34918129/ffe3b5fd-139f-4455-9216-d6e8c9672505)

I can see the Vercel preview now! 😄 I tested this query string in the preview: `comparator?repo=facebook%2Fdocusaurus&to=latest&from=v1.0.3`

The screenshot shows that there are 🐛 emojis in the headings. I was expecting to see only normal text in the headings, after your changes.

Reply:
I'm afraid those headings are part of the release description. For those, I was forced to convert them to actual emojis, since they won't affect the sorting of the releases themselves.

For actual release titles, the emojis were stripped to help with making the grouping and sorting homogeneous.

Reply:
The emoji "pushes" the heading's text so it lines up with the list items below the heading. This looks bad. 😄 

If getting rid of the emojis is too much work, how about _moving_ the emojis. For example:

```
Input: ### 🐛 Bug Fix
Output: ### Bug fix 🐛 
```

If getting rid of or moving the emojis is too hard, we _display_ the emojis. In that case you can resolve my review comments and merge. 😄

Reply:
Any sort of manipulation of the text within the release description is really complex. The descriptions are transformed between AST, markdown, React and HTML along the way. For stripping or moving those emojis, we need to manipulate such content, which is not trivial. All the current manipulations (e.g., GitHub links, converting emoji codes to actual emojis, code snippets formatting) are done automatically with remark plugins.

If we wanted to remove or move the emojis from the headings within a description, we would have to build our own plugin, which requires a huge effort. I'd leave it like this, and perhaps create a low priority ticket to try manipulating the description in the future to remove those remaining emojis from description headings.

Reply:
Let's keep things simple then, and merge what we have now.

I'll let you create the low effort issue, if you want.